### PR TITLE
Fix missing/wrong licenses for web console

### PR DESCRIPTION
**Pull request by jihoonson**

### CODE REVIEW

File: LICENSE.BINARY [hunk 1]
```diff
@@ -794,176 +794,170 @@ BINARY/EXTENSIONS/druid-kafka-eight
 BINARY/WEB-CONSOLE
 The following dependency names are NPM package names (https://www.npmjs.com).
 
-    This product bundles @babel/runtime version 7.3.4, copyright Sebastian McKenzie and other contributors,
+    This product bundles @babel/runtime version 7.4.3, copyright Sebastian McKenzie,
      which is available under an MIT license. For details, see licenses/bin/@babel-runtime.MIT.
 
+    This product bundles @types/dom4 version 2.0.1, copyright ,
```
Comment (gianm) [inline LICENSE.BINARY]: There should be a copyright holder here. If the holder is unknown or generic, then remove the `copyright ,` string.

File: licenses/bin/value-equal.MIT [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+# value-equal [![Travis][build-badge]][build] [![npm package][npm-badge]][npm]
```
Comment (gianm) [inline licenses/bin/value-equal.MIT]: This looks like a readme got in here instead of a license file.

File: licenses/bin/toggle-selection.MIT [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Toggle Selection
```
Comment (gianm) [inline licenses/bin/toggle-selection.MIT]: Should be a license file, but this looks like a readme.

File: LICENSE.BINARY (hunk 1 above)
Comment (jihoonson) [reply LICENSE.BINARY]: Thanks, fixed others as well.

File: licenses/bin/value-equal.MIT (hunk 2 above)
Comment (jihoonson) [reply licenses/bin/value-equal.MIT]: Thanks, fixed.

**Pull request merged**